internal/vds/vds.cpp

```diff
@@ -298,9 +298,7 @@ struct response metadata(
     return to_response(meta);
 }
 
-struct response handle_error(
-    const std::exception& e
-) {
+struct response to_response(std::exception const& e) {
```

Comment:
So we have two functions with the same name that differentiate by args. I assume that is intended?
If so, should these two be both on the top of the file and near each other?

Reply:
That was intended here. Moving the functions to the top off the file makes sense. :+1:
I would rename it if we decide that `to_response` is too vague.